tasks/57_pig-latin/pig-latin.js

```diff
@@ -0,0 +1,25 @@
+
+
+export function translatePigLatin(strInput) {
+  let ret = strInput.split('');
+  let suffix = [];
+
+
+  while (ret[0] !== 'a' &&
```

Comment:
1. extract condition to function named `isVowel()`
2. Inside function you can `return ['a', 'e', 'i', 'o', 'u'].includes(value)`